shopify/product-reviews-template/api/actions/sendReviewRequests.js

```diff
@@ -0,0 +1,47 @@
+import { SendReviewRequestsGlobalActionContext } from "gadget-server";
+import { v4 as uuidv4 } from "uuid";
+
+/**
+ * @param { SendReviewRequestsGlobalActionContext } context
+ */
+export async function run({ params, logger, api, connections }) {
+  let orders = await api.shopifyOrder.findMany({
+    first: 250,
+    filter: {
+      fulfillmentStatus: {
+        equals: "fulfilled",
+      },
+      requestReviewAfter: {
+        lessThanOrEqual: new Date(),
+      },
+    },
+    select: {
+      singleUseCode: true,
+      customer: {
+        email: true,
+      },
+    },
+  });
+
+  let allOrders = orders;
+
+  while (orders.hasNextPage) {
+    orders = await orders.nextPage();
+    allOrders = allOrders.concat(orders);
+  }
+
+  const options = {
```

Comment:
It seems like this could just be defined in a separate file instead of passed around as a param. We don't seem to modify it?

Reply:
Yeah, good point